website/source/docs/other/environmental-variables.html.md

```diff
@@ -185,9 +185,14 @@ required.
 
 ## `VAGRANT_PREFER_SYSTEM_BIN`
 
+This currently only applies to Windows systems.
+
 If this is set, Vagrant will prefer using utility executables (like `ssh` and `rsync`)
 from the local system instead of those vendored within the Vagrant installation.
-This currently only applies to Windows systems.
+
+Vagrant will default to using a system provided `ssh` on windows by default. This
```

Comment:
Vagrant will default to using a system provided `ssh` on windows ~~by default~~.